StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/DocumentationResources.de-DE.resx

```diff
@@ -129,6 +129,9 @@
   <data name="NonPrivateConstructorStandardTextSecondPart" xml:space="preserve">
     <value> {0}</value>
   </data>
+  <data name="ParameterNotUsed" xml:space="preserve">
+    <value>Dieser Parameter wird nicht verwendet.</value>
```

Comment:
@sharwell In German all nouns are capitalized. (I'm a German native speaker myself)